site/content/contribute/mobile/developer-setup/_index.md

```diff
@@ -17,6 +17,11 @@ A macOS computer is required to build the Mattermost iOS mobile app.
 
 Install the following prerequisite software to develop and build the iOS or Android apps. For macOS, we recommend using [Homebrew](https://brew.sh/) as a package manager.
 
+#### Install Cygwin (Only for Windows!!!)
```

Comment:
```suggestion
#### Install Cygwin (Windows only)
```

Reply:
Are there options to install Cygwin for other versions of Windows? If not, one option is to rename this header "Install Cygwin for Windows 10" and then remove the "Windows 10" subheading.

LMK if that makes sense. :)

Reply:
Hi, thank you for your kind suggestions :D I've searched and found this answer "Windows Vista, Windows Server 2008 and all later versions of Windows up to Windows 10 and Windows Server 2016." from https://cygwin.com/faq/faq.html#faq.what.supported

Reply:
Brilliant, thank you! What are your thoughts on adding that under the "Install Cygwin" heading? Then the Windows 10 heading will be more targeted.

Reply:
@justinegeffen Thanks for your suggestion. I will try to put it under the heading.